lib/DockerWrapper.js

```diff
@@ -7,6 +7,9 @@ const path = require('path');
 const uuidv1 = require('uuid/v1')
 const slugify = require('slugify');
 const tar = require('tar-fs');
+const moment = require('moment');
+const App = require('../server/models').App;
```

Comment:
Guessing `App` was an import during the coding process that never got used. Looks like you can now remove it?

Reply:
Yes, thank you!